framework/analytics/__init__.py

```diff
@@ -72,6 +75,8 @@ def build_page(rex, kwargs):
         return None
 
 
+@run_postcommit(once_per_request=False, celery=True)
```

Comment:
We removed this due to lack of request context. I reckon that's why there's a merge conflict.